bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -31,6 +31,7 @@ class Neuron(CMakePackage):
     patch("patch-v800-cmake-nvhpc.patch", when="@8.0.0%nvhpc^cmake@3.20:")
 
     version("develop", branch="master")
+    version("9.0.a5", commit="bea7eb8")
```

Comment:
Just to flag this should not be merged as-is because this commit is not on master.